auto-value-parcel/src/main/java/com/ryanharter/auto/value/parcel/AutoValueParcelExtension.java

```diff
@@ -137,6 +152,41 @@ private static boolean needsContentDescriptor(Context context) {
     return true;
   }
 
+  private static boolean needsWriteToParcel(Context context) {
```

Comment:
I would name these two methods 'has' not 'needs' since you're not longer conditionally generating. Plus is eliminates the need to do the not in the calling code.